packages/starlight/components/SiteTitle.astro

```diff
@@ -17,9 +21,15 @@ if (config.logo) {
   }
   if (err) throw new Error(err);
 }
+
+const { locale } = Astro.props
+
+const href = locale
+  ? `${import.meta.env.BASE_URL.replace(/\/$/, '')}/${locale}/`
```

Comment:
Thanks @BryceRussell!

Did you find `BASE_URL` sometimes _doesn't_ have a trailing slash? I think I may have assumed it always does elsewhere in the code base, so could be worth double checking.

Reply:
Did a quick investigation. Would probably be good to refactor everywhere we use `import.meta.env.BASE_URL` to use a helper like `withBase` so we’re consistent, i.e. this would become something like `withBase(locale ? '/locale/' : '/')`.

We can do that in a follow-up PR though. Thanks for this @BryceRussell!